src/variables.jl

```diff
@@ -63,7 +63,7 @@ lower bound on the same variable. Moreover, if the user adds a
 bridged into a supported constraint.
 """
 function supports_add_constrained_variable(model::ModelLike,
-                                       S::Type{<:AbstractScalarSet})
+                                       S::Type{<:AbstractSet})
```

Comment:
I would not change this. There is `supports_add_constrained_variables` below for vector sets.

Reply:
I agree with that, I forgot I changed it. Then, in [`node(b::LazyBridgeOptimizer, S::Type{<:MOI.AbstractSet})`](https://github.com/ilancoulon/MathOptInterface.jl/blob/1dec6d2092ad5b8408a77a0efcd1185dcd3cacd5/src/Bridges/lazy_bridge_optimizer.jl#L116), should I call `supports_add_constrained_variables` if S is an `AbstractvectorSet`, or should I write something like this in `variables.jl`?:
`supports_add_constrained_variable(model::ModelLike, S::Type{<:AbstractVectorSet}) = supports_add_constrained_variables(model, S)`

The real question could be: could we use only one fonction with two methods instead of two functions?

Reply:
I did the latter in the last commits, just for the tests to pass.

Reply:
You should do the former :)

Reply:
Ahah I changed it and filtered in `node()`.